docs/babel.rst

```diff
@@ -65,3 +65,40 @@ These are all the files you need in order to use `pyhf xml2json <cli.html#pyhf-x
 which will read all of the XML files and load the histogram data from the histogram cache.
 
 The ``HistFactory`` workspace in ``results/`` contains all of the information necessary to rebuild the XML files again. For debugging purposes, the ``pyhf`` developers will often ask for your workspace file, which means ``results/3b_tag21.2.27-1_RW_ExpSyst_36100_multibin_bkg/Excl_combined_DefaultMeasurement.root``. If you want to generate the XML, you can open this file in ``ROOT`` and run ``DefaultMeasurement->PrintXML()`` which puts all of the XML files into the current directory you are in.
+
+
+TRExFitter
+----------
+
+.. note::
+
+    For more details on this section, please refer to the ATLAS-internal `TRExFitter documentation <https://trexfitter-docs.web.cern.ch/trexfitter-docs/advanced_topics/pyhf/>`_.
+
+In order to go from ``TRExFitter`` to ``pyhf``, the good news is that the RooWorkspace files (``XML`` and ``ROOT``) are already made for you. For a given configurastion which looks like
+
+.. code:: yaml
+
+    Job: "pyhf_example"
+    Label: "..."
+
+You can expect some files to be made like so:
```

Comment:
I think for a `TRExFitter` user this is clear already, but to be very explicit maybe it is a good idea to put 

> (after the `n`/`h` and `w` steps)

after "so". The histograms and xml files are produced at different stages in `TRExFitter` and a user may wonder why they see the histograms already and not the `xml` files yet, this would clarify that they need to run an additional step in that case.
This is a minor point however, since they need to run those steps for a fit with `TRExFitter` anyway, so it is very unlikely they would run into this issue.